Sandbox/Windows10/HelloWorld/HelloWorld.Unity/App.xaml.cs

```diff
@@ -25,30 +25,27 @@ namespace Sample
 {
     sealed partial class App : PrismApplication
     {
-        public static IPlatformNavigationService NavigationService { get; private set; }
-
         public App()
         {
             InitializeComponent();
         }
 
-        public override void RegisterTypes(IContainerRegistry container)
+        protected override void RegisterTypes(IContainerRegistry container)
         {
-            container.RegisterForNavigation<MainPage, MainPageViewModel>(nameof(MainPage));
-            container.RegisterForNavigation<ItemPage, ItemPageViewModel>(nameof(ItemPage));
+            container.RegisterForNavigation<MainPage, MainPageViewModel>();
+            container.RegisterForNavigation<ItemPage, ItemPageViewModel>();
         }
 
-        public override void OnInitialized()
+        protected override void OnInitialized()
         {
-            NavigationService = Prism.Navigation.NavigationService.Create(Gestures.Back, Gestures.Forward, Gestures.Refresh);
             NavigationService.SetAsWindowContent(Window.Current, true);
         }
 
-        public override void OnStart(StartArgs args)
+        protected override void OnStart(StartArgs args)
         {
             if (args.StartKind == StartKinds.Launch)
             {
-                NavigationService.NavigateAsync(nameof(MainPage));
+                NavigationService.NavigateAsync("/MainPage");
```

Comment:
can't we use $"/{nameof(MainPage)}"? I don't like suggesting magic strings

Reply:
two things here... 1) using name of a View is actually worse IMO... 2) remember this is a Sandbox app